Manual.md

```diff
@@ -127,14 +127,16 @@ exception, start a PR and make an argument for why that particular piece of
 software, while not meeting any of the following requirements, is a good candidate for
 the Void packages system.
 
-1. System: The software should be installed system-wide, not per-user.
+1. System: The software is best suited to system-wide, rather than per-user, installation.
 
 1. Compiled: The software needs to be compiled before being used, even if it is
    software that is not needed by the whole system.
 
 1. Required: Another package either within the repository or pending inclusion
    requires the package.
 
+In particular, new themes and fonts are highly unlikely to be accepted.
```

Comment:
Also small shell scripts. Fonts are welcome if they provide value beyond aesthethic, eg. contains glyphs for script missing in already packaged fonts.